js/main.js

```diff
@@ -1,2 +1,47 @@
 /* global data */
 /* exported data */
+
+var $url = document.querySelector('.image');
+var $urlInput = document.querySelector('#url');
+
+function changeImage(event) {
+  if ($urlInput.value === '') {
+    $url.src = 'images/placeholder-image-square.jpg';
+  } else {
+    $url.src = $urlInput.value;
+  }
+}
+
+$urlInput.addEventListener('input', changeImage);
+
+var $form = document.querySelector('.form');
+var $titleInput = document.querySelector('#title');
+var $notesInput = document.querySelector('#notes');
+
+function onSubmit(event) {
+  var object = {};
+  object['image URL'] = $urlInput.value;
+  object['entry title'] = $titleInput.value;
+  object['entry notes'] = $notesInput.value;
+
+  if (Boolean(localStorage.getItem('data')) === true) {
```

Comment:
I would save the value returned from `getItem` in a variable, so you don't need to call it here, as well as on the next line.